queries.sql

```diff
@@ -111,4 +111,42 @@ GROUP BY  species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE  date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
```

Comment:
- [ ] The correct output for the second query `List of all animals that are pokemon (their type is Pokemon).` should be as follows:

Expected output
![image](https://github.com/waelelsafty07/vet/assets/81932478/b574424e-27f0-4674-914e-62d54fa0ee53)

Your Output
![image](https://github.com/waelelsafty07/vet/assets/81932478/5a3811ab-7052-44b1-96ce-9b84fcf3f3ab)

Kindly fix your query so as to produce the correct result

### Note that the screenshot will have to be updated